src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx

```diff
@@ -44,7 +45,11 @@ const MhvSecondaryNavItem = ({
 
   return (
     <div key={key} className={itemClass} data-testid="mhv-sec-nav-item">
-      <a href={href} className="vads-u-text-decoration--none">
+      <a
+        href={href}
+        data-dd-action-name={actionName}
```

Comment:
Should there be a test to assert this html property?

Reply:
Added a test, but I may be missing something obvious about how to query the attribute using testing-library, so I have `result.getAttribute('data-dd-action-name')` in there, which may or may not be okay… `expect(result).to.have.attribute('data-dd-action-name', actionName)` wasn't working

Reply:
That's the way I've done it in [other PRs](https://github.com/department-of-veterans-affairs/vets-website/pull/29583/files#diff-3d59f62b71da7450558604d2c5e339b621022381d30d42ccfe5e9adfb1ea4b76R13-R14).

Reply:
Figuring out exactly what you have to say to chai to get what you want is tricky. You might want `.to.have.property()`.

https://www.chaijs.com/guide/styles/

Reply:
I tried `to.have.property()` and it didn't work. Guessing bc an HTMLElement's attributes are not quite the same as props